src/citrine/informatics/predictor_evaluation_result.py

```diff
@@ -220,3 +221,41 @@ def responses(self) -> Set[str]:
     def metrics(self) -> Set[PredictorEvaluationMetric]:
         """:Set[PredictorEvaluationMetric]: Metrics for which results are present."""
         return self._evaluator.metrics
+
+
+class HoldoutSetResult(Serializable["HoldoutSetResult"], PredictorEvaluationResult):
+    """Result of performing holdout evaluation on a predictor.
+
+    Results held-out response can be accessed via ``result['response_name']``,
+    where ``result`` is a
+    :class:`citrine.informatics.predictor_evaluation_result.HoldoutSetResult`
+    and ``'response_name'`` is a response analyzed by a
+    :class:`citrine.informatics.predictor_evaluator.HoldoutSetEvaluator`.
+
+    """
+
+    _evaluator = properties.Object(PredictorEvaluator, "evaluator")
```

Comment:
Should the evaluator be specific as a `HoldoutSetEvaluator` here ? Same goes for the CV result above.

I think the backend has these typed specifically. but I'm unsure if its straightforward to check that upon deserialization here.

Reply:
I made it more specific. But also, the `type` field indicates whether it's a `HoldoutSetEvaluator` or a `CrossValidationEvaluator`.